src/components/sectioningConfirm/SectioningConfirm.tsx

```diff
@@ -41,23 +43,48 @@ export default function SectioningConfirm({
 }: SectioningConfirmProps) {
   const [current, send, service] = useMachine(createSectioningConfirmMachine());
 
+  /**Get plans in the same order as displayed*/
```

Comment:
Could you explain this change to me please?

Reply:
The labwares need to be displayed in Store page in the same order as displayed in Sectioning Confirm page. The order in SectioningConfirm page is - all TUBEs first followed by groups of Slide or Visium labware. So, this will make sure the plans are in the same order (as Confirm page) before navigating to Store page. Hope that makes sense.